test/Test/Database/LSMTree/Normal/StateMachine/DL.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE OverloadedLists #-}
```

Comment:
Where is that used?

Reply:
Nothing! Forgot to remove it after I made some other changes